applesingle.c

```diff
@@ -1278,7 +1278,7 @@ int main(int argc, char * argv[])
 "Encoding of files is not supported on this platform.\n"
 "\n"
 #endif
-, me, me, me, me, me);
+, me, me, me);
```

Comment:
A compiler warning is expected here when !defined(ENABLE_ENCODING). Your patch would introduce a real bug when defined(ENABLE_ENCODING).

Reply:
yeah this was mostly just a quick hacky fix because it was saying that there were too many parameters. maybe this part of the printf call can also be put inside the `#ifdef`'s so it uses a different # of parameters based on the value of defined(ENABLE_ENCODING)?

Reply:
Okay. I've committed a change to along those lines to avoid the warning when !defined(ENABLE_ENCODING).